api/hubdb.ts

```diff
@@ -11,6 +11,14 @@ import {
 
 const HUBDB_API_PATH = 'cms/v3/hubdb';
 
+export function listTables(
+  accountId: number
+): AxiosPromise<Table> {
```

Comment:
The response type here is actually going to look a little different. There's an overview of it [here](https://developers.hubspot.com/beta-docs/reference/api/cms/hubdb#get-%2Fcms%2Fv3%2Fhubdb%2Ftables), but it'll look similar to the existing `FetchRowsResponse` type.

Reply:
Sorry you're definitely right, forgot to add that type. Thanks!